bundle/tests/scorecard/kuttl/day2operation/03-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: rc.app.stacks/v1beta2
+kind: RuntimeOperation
+metadata:
+  name: example-runtime-operation-1
+spec:
+  podName: day2-operation-rc-0
+  containerName: app
+status:
+  conditions:
+    - status: 'False'
```

Comment:
This could be enhanced to include `type` and `reason`

```
status:
  conditions:
    - lastUpdateTime: '2021-11-17T13:22:16Z'
      message: Failed to find pod 'Specify_Pod_Name_Here' in namespace 'demo'
      reason: Error
      status: 'False'
      type: Started
```